src/graphic/helper/parseText.ts

```diff
@@ -678,7 +689,6 @@ function wrapText(
                     // Append lastWord if have
                     if (currentWord) {
                         line += currentWord;
-                        accumWidth += currentWordWidth;
```

Comment:
Each character's width has already added to `accumWidth` when looping through. Typical case:

**Before**
![image](https://user-images.githubusercontent.com/779050/143553639-5ad3dd51-c184-4518-be62-c4a14db457d9.png)
The third line's width is calculated wrong due to this line.

**After**
![image](https://user-images.githubusercontent.com/779050/143553697-5d575ea4-8234-4d22-850d-8647c46ed5cd.png)